index-test.js

```diff
@@ -1,50 +1,71 @@
 // MINIMUM requirement of Unit Tests (as stated on project brief)
 
-setTimeout(() => { //Allows time for the elements to appear on the page as they're not present onload.
-
-  // CHECKBOX TESTING
-
-  test("Checking first item marks it as complete", () => {
-  const checkbox = document.querySelectorAll("input[type='checkbox']")
-  const title = document.querySelectorAll('.movie-title');
-  checkbox[0].click();
-  const result = title[0].classList.contains('strikethrough'); 
-  const expected = true;
-  equal(result, expected)
-  checkbox[0].click(); //undoing the click
-  });
-
-  test("Checking third item marks it as complete", () => {
-  const checkbox = document.querySelectorAll("input[type='checkbox']")
-  const title = document.querySelectorAll('.movie-title');
-  checkbox[2].click();
-  const result = title[2].classList.contains('strikethrough'); 
-  const expected = true;
-  equal(result, expected)
-  checkbox[2].click(); //undoing the click
-  });
-
-  test("Checking the movie is not checked on default", () => {
-  const title = document.querySelectorAll('.movie-title');
-  const result2 = title[0].classList.contains('strikethrough'); 
-  const result1 = title[2].classList.contains('strikethrough'); 
-  const expected = false;
-  equal(result1, expected, 'first item tested')
-  equal(result2, expected, 'third item tested')
-  });
-
-  test("Checking unchecking removes the strike", () => {
-  const checkbox = document.querySelector("input[type='checkbox']")
-  const title = document.querySelector('.movie-title');
-  checkbox.click();
-  checkbox.click();
-  const result = title.classList.contains('strikethrough'); 
-  const expected = false;
-  equal(result, expected)
-  });
+ // CHECKBOX TESTING
+
+test("Checking first item marks it as complete", () => {
```

Comment:
Brilliant set up on these tests ✨ 

I find the step comments really helpful to read 😸